### PR TITLE
(feat) Added angular ui.router / added routes / changed app name to wildDonut i...

### PR DESCRIPTION
**Pull request by sidewalksalsa**
...n index.html

### CODE REVIEW

**Comment by sidewalksalsa** [comment]
@benignbemine would you be so kind to review?

File: client/www/app/user/user-settings/user-settings.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
+<script>
```
Comment (benignbemine) [inline client/www/app/user/user-settings/user-settings.html]: this should be in a userSettingsController.js file

File: client/www/app/app.js [hunk 2]
```diff
@@ -3,7 +3,65 @@
...
-angular.module('starter', ['ionic'])
+var app = angular.module('wildDonut', ['ionic', 'ui.router']);
+
+app.config(function($stateProvider, $urlRouterProvider) {
+  // for any unmatched urls, redirect to /home
+  $urlRouterProvider.otherwise('/');
+
+  // full routing for app
+  $stateProvider
+    .state('signIn', {
```
Comment (benignbemine) [inline client/www/app/app.js]: signIn or logIn?

File: client/www/app/app.js [hunk 3]
```diff
@@ -3,7 +3,65 @@
 // angular.module is a global place for creating, registering and retrieving Angular modules
 // 'starter' is the name of this angular module example (also set in a <body> attribute in index.html)
 // the 2nd parameter is an array of 'requires'
-angular.module('starter', ['ionic'])
+var app = angular.module('wildDonut', ['ionic', 'ui.router']);
```
Comment (benignbemine) [inline client/www/app/app.js]: we can refactor this to more closely follow john papa's styleguide

**Pull request merged**